wrapper/schemaless_test.go

```diff
@@ -0,0 +1,76 @@
+package wrapper_test
+
+import (
+	"testing"
+	"time"
+	"unsafe"
+
+	"github.com/taosdata/driver-go/v2/errors"
+	"github.com/taosdata/driver-go/v2/wrapper"
+)
+
+var conn unsafe.Pointer
+
+func TestMain(m *testing.M) {
+	var err error
+	conn, err = wrapper.TaosConnect("", "root", "taosdata", "log", 0)
+	if err != nil {
+		panic(err)
+		return
+	}
+	m.Run()
+	defer wrapper.TaosClose(conn)
+}
+
+func BenchmarkTelnetSchemaless(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		result := wrapper.TaosSchemalessInsert(conn, []string{
+			"sys_if_bytes_out 1636626444 1.3E3 host=web01 interface=eth0",
+		}, wrapper.OpenTSDBTelnetLineProtocol, "")
+		code := wrapper.TaosError(result)
+		if code != 0 {
+			errStr := wrapper.TaosErrorStr(result)
+			wrapper.TaosFreeResult(result)
+			b.Error(&errors.TaosError{
+				Code:   int32(code) & 0xffff,
```

Comment:
does the mask potentially make the error code be mistake read (in future??

Reply:
I've added NewError function in errors package and updated test code.